src/pages/dar_application/ProgressReportApplication.tsx

```diff
@@ -23,11 +23,13 @@ type ProgressReportApplicationProps = {
   readonly history: History;
   readonly location?: Location;
   readonly researcher: DuosUser;
+  readonly countriesOfOperation: string[]
 };
 
-export const ProgressReportApplication = ({ dar, datasets, readOnlyMode = true, history, location, researcher }: ProgressReportApplicationProps) => {
+export const ProgressReportApplication = ({ dar, datasets, readOnlyMode = true, history, location, researcher, countriesOfOperation }: ProgressReportApplicationProps) => {
     const initialState = {
         ...dar,
+        ...dar.data,
```

Comment:
We need to spread dar.data on this object if we intend to use it as typed because that's where it is stored in the server response.

Reply:
I think we either send `dar` or `dar.data` in the DAR page.